GarbageCan.Application.UnitTests/Boosters/Commands/UpdateBoosterCycleCommandTests.cs

```diff
@@ -169,7 +169,7 @@ public async Task ShouldActivateQueuedBooster_IfSlotBecomesAvailable()
                 new()
                 {
                     GuildId = guildId,
-                    ExpirationDate = now.Add(TimeSpan.FromSeconds(5)),
+                    ExpirationDate = now.ToUniversalTime().Subtract(TimeSpan.FromSeconds(5)),
```

Comment:
a potentially better idea is calling touniversal at the declaration  of the variable

Reply:
Ya, that will work for the tests, but an even better idea would be to not ever call this at all and take whatever the `IDateTime` gives us as Now. This way in our implementation, we can just call it once all of the consuming code would not have to worry about the time conversion. However, that's much more of a change than just updating this test.